docs/docs/services.md

```diff
@@ -725,3 +725,300 @@ const createPost = (input) => {
 ```
 
 This makes sure that the user that's logged in and creating the post cannot reuse the same blog post title as one of their own posts.
+
+## Caching
+
+Redwood provides a simple [LRU cache](https://www.baeldung.com/java-lru-cache) for your services. With an LRU cache you never need to worry about manually expiring or updating cache items. You either read an existing item (if its **key** is found) or create a new cached item if it isn't. This means that over time the cache will get bigger and bigger until it hits a memory or disk usage limit, but you don't care: the cache software is responsible for removing the oldest/least used members to make more room. For many applications, its entire database resultset may fit in cache!
```

Comment:
```suggestion
Redwood provides a simple [LRU cache](https://www.baeldung.com/java-lru-cache) for your services. With an LRU cache you never need to worry about manually expiring or updating cache items. You either read an existing item (if it's **key** is found) or create a new cached item if it isn't. This means that over time the cache will get bigger and bigger until it hits a memory or disk usage limit, but you don't care: the cache software is responsible for removing the oldest/least used members to make more room. For many applications, its entire database resultset may fit in cache!
```

Reply:
Hmm, I think my usage is correct: "it's" means "it is" and you wouldn't say "if it is key is found", right?

Reply:
My bad you're right.